Add readmes to notebooks (#184)
openvinotoolkit/openvino_notebooks: notebooks/001-hello-world/README.md

# Introduction to OpenVINO

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F001-hello-world%2F001-hello-world.ipynb)

![coco image](data/classification.jpg)

Basic introduction that shows basic operations using OpenVINO, including inference on a given IR model.

> ## Content table

Reviewer: ```suggestion
## Notebook Contents
```

> Basic introduction that shows basic operations using OpenVINO, including inference on a given IR model.

Reviewer 2: ```suggestion
This notebook shows how to do inference on an OpenVINO IR model.
```
